packages/cloudwatch-agent/Cargo.toml

```
@@ -0,0 +1,21 @@
+[package]
+name = "cloudwatch-agent"
+version = "0.1.0"
+edition = "2021"
+publish = false
+build = "build.rs"
+
+[lib]
+path = "pkg.rs"
```

After rebasing #3288, please make this change and delete `pkg.rs`. The rebase will not pick this up, nor will the build fail as a result of the rebase. Thanks!

```suggestion
path = "../packages.rs"
```